lib/tasks/verify.js

```diff
@@ -60,17 +62,37 @@ function pathsToGlob(paths) {
 }
 
 /**
+ * Custom fail reporter that doesn't fail on Scss Lint warnings
+ */
+function failReporter() {
+	var stream = through.obj(function(file, enc, cb) {
+		var error;
+		if (file.scsslint.errors > 0) {
+			error = new gutil.PluginError('gulp-scss-lint', {
+				message: 'ScssLint failed for: ' + file.relative,
```

Comment:
Actually, that line of code is copied from the fail reporter gulp-scsslint offers. But I will update anyway